packages/docs/docs/docs/deploying.md

```diff
@@ -178,6 +178,16 @@ to understand is that the contracts deployed by the `push` command are logic con
 upgradeable instances, as we will see later in the
 [Upgrading your project](https://docs.zeppelinos.org/docs/upgrading.html) section.
 
+For example, you wouldn't interact with the instance of MyContract that we just deployed. This instance is only intended for defining the behavior or logic of the contract. You can find the address of this logic contract in the `zos.dev-<network_id>.json` file, under `contracts.MyContract.address`. Remember to never interact with this address!
+
+So what _do_ you interact with? As we'll see in depth in the next guide, what you do is create a proxy for the logic contract, and interact with this proxy contract instead. 
```

Comment:
Note that in the following guide we hardly ever use the term "proxy". I'd refer to them as "upgradeable instances" for consistency.